lang/cs/Org.Apache.REEF.Client/app.config

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There are a few app.config files that are being added - is this required? We have a general app.config that gets applied to all projects. Plus the bridge has one that is generated via code. So I think these can be removed.

Reply:
I will check that. It is automatically added by Visual Studio.